picard/util/tags.py

```diff
@@ -58,6 +59,7 @@
 
 
 def _markdown(text: str):
+    text = html.escape(text)
```

Comment:
Shouldn't we escape only when using html (and not markdown)?

Reply:
Not sure, as you can actually have HTML in Markdown it probably is correct that way.

Reply:
I'm confused.  Don't we want to escape any reserved HTML characters in the text before we pass it to `markdown` to turn the markdown text into html?  If we did the `html.escape()` after the markdown processing, wouldn't that destroy the resulting html text produced by `markdown`?

Reply:
My previous comment applies to the tags.  With respect to the function documentation, I'll need to go back into the code to see how it is generated.  I assumed (and I should know better) that the proper encoding and checks had already been addressed, so I just used the existing code for the content of the "Functions" tab.